flex/views/experiment/TabSampleSetupView.mxml

```diff
@@ -2284,9 +2199,9 @@
 			<mx:SetProperty target="{label8}" name="width" value="340"/>
 			<mx:SetProperty target="{label7}" name="width" value="340"/>
 			<mx:SetProperty target="{label2}" name="width" value="340"/>
-			<mx:SetProperty target="{text1}" name="width" value="340"/>
-			<mx:SetProperty target="{samplePoolingLabel}" name="width" value="340"/>
-			<mx:SetProperty target="{samplesToBePreppedCheckbox}" name="selected" value="false"/>
+			<mx:SetProperty target="{parentDocument.seqSetupView.text1}" name="width" value="340"/>
+			<mx:SetProperty target="{parentDocument.seqSetupView.samplePoolingLabel}" name="width" value="340"/>
+			<mx:SetProperty target="{parentDocument.seqSetupView.samplesToBePreppedCheckbox}" name="selected" value="false"/>

```

Comment:
Should we be setting properties of elements that are on the seqSetupView tab in this sampleSetupView tab?

Maybe this should be moved to the experiment edit view?  I believe it has the notion of state that you could pick up.  Let me know if I don't fully understand.